template/src/store/reducers/loading.js

```diff
@@ -0,0 +1,18 @@
+const loadingReducer = (state = {}, action) => {
+  const {type} = action
+  // remove module name when create from createSlice
+  // cause by create type by createSlice it auto add prefix module
```

Comment:
please remove this comment if not neccessary

Reply:
I think we need to keep this
In this comment i explained why i have to remove some text from type
In case someone want to bring this reducer into another project that not using redux toolkit, they will know to remove some code to make this works